Servers/WindowServer/WSWindowManager.cpp

```diff
@@ -67,29 +65,22 @@ WSWindowManager::WSWindowManager()
         m_system_menu->add_item(make<WSMenuItem>(*m_system_menu, 300, "Shutdown..."));
         m_system_menu->on_item_activation = [this, apps](WSMenuItem& item) {
             if (item.identifier() >= 1 && item.identifier() <= 1u + apps.size() - 1) {
-                if (fork() == 0) {
-                    const auto& bin = apps[item.identifier() - 1].binary_name;
-                    execl(bin, bin, nullptr);
-                    ASSERT_NOT_REACHED();
-                }
+                CProcess::start_detached(apps[item.identifier() - 1].binary_name);
             }
             switch (item.identifier()) {
             case 100:
                 reload_config(true);
                 break;
             case 200:
-                if (fork() == 0) {
-                    execl("/bin/About", "/bin/About", nullptr);
-                    ASSERT_NOT_REACHED();
-                }
+                CProcess::start_detached("/bin/About");
                 return;
-            case 300:
-                if (fork() == 0) {
-                    execl("/bin/shutdown", "/bin/shutdown", "-n", nullptr);
-                    ASSERT_NOT_REACHED();
-                }
+            case 300: {
+                Vector<StringView> args;
+                args.append("-n");
+                CProcess::start_detached("/bin/shutdown", args);
```

Comment:
We should have a way to construct a `Vector` from an `initializer_list` :)
Then we could write:
```cpp
CProcess::start_detached("/bin/shutdown", { "-n" });
```

Reply:
I've added that mechanism to `Vector`.